pom.xml

```diff
@@ -844,6 +861,7 @@
             <id>tc</id>
             <properties>
                 <skipUnitTests>true</skipUnitTests>
+                <postgresql.server.version>9.6</postgresql.server.version>
```

Comment:
Should this postgres server version be moved up to be near the client version (around line 30)?

Reply:
Preferably, yes. But I didn't want to bother others, so I left it in the profile. Would you prefer it in the upper place, too?

Reply:
Hmm. Well, I don't want it in two places, if that's what you mean. But sure, if it can be in a single place, I'd prefer the single place to be at the top of the file with the other properties. (I just tried this locally and it seemed to work.). With the settings side by side, it's a little confusing what's what:

        <postgresql.server.version>9.6</postgresql.server.version>
        <postgresql.version>42.2.18</postgresql.version>

Maybe at some point we should rename the "42" one to something more specific.

Reply:
OK if it confuses people, maybe we should leave it in the profile then. It's only used with testcontainers, so it would make sense to leave it there.

Reply:
That's fine.